modules/distribution/product/src/main/conf/deployment.toml

```diff
@@ -146,7 +146,7 @@ properties.'worker.thread.count' = 1
 [apim.cors]
 allow_origins = "*"
 allow_methods = ["GET","PUT","POST","DELETE","PATCH","OPTIONS"]
-allow_headers = ["authorization","Access-Control-Allow-Origin","Content-Type","SOAPAction","apikey", "testKey"]
+allow_headers = ["authorization","Access-Control-Allow-Origin","Content-Type","SOAPAction","apikey","Internal-Key"]
```

Comment:
Do we need to allow this header anymore, since we use the `bearer` token for internal testing as well?

Reply:
sorry i think it's `Internal-Key` header not the regular authorization header

Reply:
No, as I discussed with Tharindu, we can't use the Authorization header. We have to use the Internal-Key header so that anybody can write a handler and add their own logic to authenticate the request.